src/main/java/com/treasure_data/td_import/source/S3Source.java

```diff
@@ -53,20 +56,25 @@
 
         System.setProperty("com.amazonaws.sdk.disableCertChecking", "1");
 
-        AmazonS3Client client = createAmazonS3Client(desc);
+        String region = getBucketLocation(desc, bucket);
```

Comment:
We may want to add a special error message of incompatibility in case of insufficient permissions.

Reply:
Updated implementation to output log #f6d01cd
```
WARNING: Failed to get S3 bucket location. Please make sure that your account is given 's3:GetBucketLocation' role.
 57 Jul 02, 2019 1:23:59 PM com.treasure_data.td_import.Import getSources
 58 WARNING: fallback and create source as LocalFileSource: s3://<ACCESS_KEY>:<SECRET_KEY>@s3.us-east-1.amazonaws.com/bucket-name/path/to/test.csv
 59 com.amazonaws.services.s3.model.AmazonS3Exception: Access Denied (Service: Amazon S3; Status Code: 403; Error Code: AccessDenied; Request ID: 0313CCAA8F31D130    ; S3 Extended Request ID: lhNgnkNNtHQI35sJ7nGdDcJQdsqLcOqiT4SN39bRTNVdFOp70GV3f+112wiu04hvBkHlz85q5yY=), S3 Extended Request ID: lhNgnkNNtHQI35sJ7nGdDcJQdsqLc    OqiT4SN39bRTNVdFOp70GV3f+112wiu04hvBkHlz85q5yY=
 ```